From: hololeap <hololeap@protonmail.com>
Signed-off-by: hololeap <hololeap@protonmail.com>

doctest-parallel does not currently work with Setup.hs

See: <martijnbastiaan/doctest-parallel#45>

diff -urN regex-tdfa-1.3.2/regex-tdfa.cabal regex-tdfa-1.3.2-r1/regex-tdfa.cabal
--- regex-tdfa-1.3.2/regex-tdfa.cabal	2022-08-14 00:42:54.578227087 -0600
+++ regex-tdfa-1.3.2-r1/regex-tdfa.cabal	2022-08-14 00:43:17.428227098 -0600
@@ -175,6 +175,7 @@
     ghc-options:        -O2
 
 test-suite doc-test
+  buildable:      False
   type:           exitcode-stdio-1.0
   hs-source-dirs: test
   main-is:        DocTestMain.hs
